include/dynd/types/string_type.hpp

```diff
@@ -22,6 +22,8 @@ class DYND_API string : public bytes {
 
   string(const std::string &other) : string(other.data(), other.size()) {}
 
+  string(const bytes &other) : string(other.data(), other.size()) {}
+
   bool operator<(const string &rhs) const
```

Comment:
Ah, okay. There probably is a better way to do it, but I don't know what it is offhand. So let's leave this in and fix it whenever someone feels like playing around with it.